### DISCUSSION TITLE
suggestion

### DISCUSSION THREAD

**Discussion by aml2023529** [Zrips/CMI#8201] [Enhancement]
Blocking kicked out of the server command to add time for example, I want to block him thirty days after the lifting of the current configuration file is very cluttered I hope that from the confirm a good planning about the function of the switch please categorize him should belong to a directory in the confirm I want to put only the key settings such as the database settings!

**Comment by CloudeLecaw** [comment]
may you should give your suggest a title instead of "suggestion"

**Comment by CloudeLecaw** [comment]
> Blocking kicked out of the server command to add time for example, I want to block him thirty days after the lifting of the current configuration file is very cluttered I hope that from the confirm a good planning about the function of the switch please categorize him should belong to a directory in the confirm I want to put only the key settings such as the database settings!

you mean timeban? that already exists...


cmi tempban [playerName] [time] (reason) (-s)
cmi tempipban [ip/playerName] [time] (reason) (-s)

**Reply by aml2023529** [reply]
Have you considered reorganizing the commands? It's true that there are too many functions and sometimes when I need a command for many functions, I suddenly can't remember and can't find it. For example, if I need to ban a player, I type in banlist and he will list the commands such as banip Tempban and so on for me.

**Reply by aml2023529** [reply]
He has too many commands cmi list commands have to check for a very long time don't you think so for example this time I fed back the time of the ban I list also didn't see him this command to show this is the need to optimize the place I input kit he will show all the commands of the kit related functions and confirm the configuration is really very cluttered

**Reply by CloudeLecaw** [reply]
cmi checkcommand ban

will post you all related cimmands with the wird ban in it

**Reply by aml2023529** [reply]
No, you may not have understood what I was trying to say, I mean related commands like /ban this related command I /banlist he will show his related commands unban banip tempban show it is also easy to use my friend bought cmi also prefer EssentialsX he thinks he is very easy to get started I also think that cmi nowadays is just so many functions that it is hard to get started SunLight's command planning is very good he separates all the commands of a related function and related configurations what I need I go directly to the corresponding configuration file to modify the cmi. It's hard to get started SunLight's command planning is very good he separates all the commands for a related function and the related configurations what I need to do I go directly to the corresponding configuration file to modify cmi what I need to modify his configurations each time it takes a long time to turn over you don't you have a Settings file why don't you try to split it up a few more I think it's easier this way!

**Reply by aml2023529** [reply]
Settings is a file that I haven't opened since I used it a couple of times because a lot of the settings are in the confirm file and I have to dig around for a long time to find that setting every time I try to find the relevant configuration.

**Reply by aml2023529** [reply]
A lot of features is a good thing I also because I heard that my friend recommended me to use it before he still want to ask you to pay attention to some of the settings related to the more and more cumbersome situation!
